### PR TITLE
[SA] Added test for migration using Testcontainers

### PR DESCRIPTION
**Pull request by SarpongAbasimi**
What I did in this PR

- Tested the  implementation for the migration using TestContainers
- Added dependencies for implementing logging in the application
- Created a `run.sh` script to load env config into `sbt` and added it to `.gitignore`

### CODE REVIEW

File: modules/persistenceService/src/test/scala/migrations/PsqlMigrationsSpec.scala [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+    it("should be started") {
+      val driverName    = DriverName(container.driverClassName)
+      val connectionUrl = ConnectionUrl(container.jdbcUrl)
+      val user          = User(container.username)
+      val password      = PassWord(container.password)
+
+      DbMigrations
+        .migrate[IO](ApplicationConfig(driverName, connectionUrl, user, password))
+        .map(_.migrationsExecuted shouldBe (1))
+        .unsafeRunSync()
```
Comment (fedefernandez) [inline modules/persistenceService/src/test/scala/migrations/PsqlMigrationsSpec.scala]: In a following PR we could use `cats-effect-testing-scalatest` and avoid the `unsafeRunSync`

https://github.com/typelevel/cats-effect-testing#scalatest

**Review by fedefernandez — APPROVED** [review]
Looks good to me!

File: modules/persistenceService/src/test/scala/migrations/PsqlMigrationsSpec.scala (hunk 1 above)
Comment (SarpongAbasimi) [reply modules/persistenceService/src/test/scala/migrations/PsqlMigrationsSpec.scala]: I will amend it push a fix.
I guess `unsafe` is really unsafe.

Comment (fedefernandez) [reply modules/persistenceService/src/test/scala/migrations/PsqlMigrationsSpec.scala]: 😂   
IMO we should have just one single `unsafe` per test. Your code is totally fine but using scalatest will allow us to be sure there's only one in the future.

**Pull request merged**